roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -12,5 +12,12 @@
 - name: start aci-gw container
   service: name=aci-gw state=started
 
+- name: Contiv | Wait for netmaster
+  action: shell netctl tenant ls
```

Comment:
let's use the `shell` module instead, to keep it consistent with rest of playbook.